Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.h

```diff
@@ -42,6 +42,36 @@ class WebGLFramebuffer;
 class WebGLRenderingContextBase;
 struct XRWebGLLayerInit;
 
+struct WebXRExternalRenderbuffer {
+    GCGLOwnedRenderbuffer rbo;
+    GCEGLOwnedImage image;
+
+    void destroyImage(GraphicsContextGL&);
+    void release(GraphicsContextGL&);
+    void leakObject();
+};
+
+template<typename T>
+struct WebXRAttachmentSet {
+    T colorBuffer;
+    T depthStencilBuffer;
+
+    void release(GraphicsContextGL& gl)
+    {
+        colorBuffer.release(gl);
+        depthStencilBuffer.release(gl);
+    }
+
+    void leakObject()
```

Comment:
Ooh what is this for? Leak but not return seems surprising.

Reply:
It's used in `WebXROpaqueFramebuffer::~WebXROpaqueFramebuffer()` I think but would need to defer to Dan